assets/test/dependencies/Sources/main.swift

```diff
@@ -0,0 +1,3 @@
+import SwiftSyntax
```

Comment:
I know this doesn't actually affect the tests but should this import Swift Markdown instead of Swift Syntax?

Reply:
Awesome catch, I swapped libs